frontend/app/src/tx-flows/updateLeveragePosition.tsx

```diff
@@ -65,8 +65,8 @@ const stepNames: Record<Step, string> = {
   approveLst: "Approve {tokenName}",
   decreaseDeposit: "Decrease Deposit",
   increaseDeposit: "Increase Deposit",
-  leverDownTrove: "Decrease Leverage",
-  leverUpTrove: "Increase Leverage",
+  leverDownTrove: "Decrease Multiply",
+  leverUpTrove: "Increase Multiply",
```

Comment:
This sounds a bit like two contradictory verbs to me (decrease & multiply), maybe using multiplier (“Decrease multiplier”) could help?